src/support-prompt-group/__tests__/support-prompt-group.test.tsx

```diff
@@ -0,0 +1,164 @@
+
+const onItemClick = vi.fn();
+
+const defaultProps = {
+  onItemClick,
+  ariaLabel: "Test support prompt group",
+  items: [
+    {
+      text: "Item 1",
+      id: "item-1",
+    },
+    {
+      text: "Item 2",
+      id: "item-2",
+    },
+    {
+      text: "Item 3",
+      id: "item-3",
+    },
+  ],
+};
+
+export function renderSupportPromptGroup(
+  props: Partial<SupportPromptGroupProps>,
+  ref?: React.Ref<SupportPromptGroupProps.Ref>,
+) {
+  const renderResult = render(<SupportPromptGroup ref={ref} {...defaultProps} {...props} />);
+  const wrapper = createWrapper(renderResult.container).findSupportPromptGroup()!;
+
+  return wrapper;
+}
+
+describe("Support prompt group", () => {
+  test("Renders null with no items", () => {
+    const wrapper = renderSupportPromptGroup({
+      items: [],
+    });
+
+    expect(wrapper).toBeNull();
+  });
+
+  test("Finds number of items", () => {
+    const wrapper = renderSupportPromptGroup({});
+
+    expect(wrapper.findItems().length).toBe(3);
+  });
+
+  test("Finds item by id", () => {
+    const wrapper = renderSupportPromptGroup({});
+
+    expect(wrapper.findItemById("item-1")?.getElement()).toHaveTextContent("Item 1");
+  });
+
+  test("fires onClick", () => {
+    const wrapper = renderSupportPromptGroup({});
+
+    wrapper.findItemById("item-1")!.click();
+
+    expect(onItemClick).toHaveBeenCalledTimes(1);
+  });
+
+  test("Assigns vertical class with vertical alignment", () => {
+    const wrapper = renderSupportPromptGroup({
+      alignment: "vertical",
+    });
+
+    expect(wrapper.getElement()).toHaveClass(styles.vertical);
+  });
+
+  describe("a11y", () => {
+    test("Group has accessible name", () => {
+      const wrapper = renderSupportPromptGroup({});
+
+      expect(wrapper.getElement()).toHaveAccessibleName("Test support prompt group");
+    });
+
+    test("Prompt has accessible name", () => {
+      const wrapper = renderSupportPromptGroup({});
+
+      expect(wrapper.findItemById("item-1")!.getElement()).toHaveAccessibleName("Item 1");
+    });
+  });
+
+  describe("Keyboard navigation", () => {
+    afterEach(() => {
+      cleanup();
+    });
+    const ref: { current: SupportPromptGroupProps.Ref | null } = { current: null };
+
+    test("Arrow keys move focus in vertical alignment", () => {
+      const wrapper = renderSupportPromptGroup({}, ref);
+
+      ref.current?.focus("item-1");
+
+      fireEvent.keyDown(wrapper.getElement(), { keyCode: KeyCode.right });
+      expect(wrapper.findItemById("item-2")!.getElement()).toHaveFocus();
+
+      fireEvent.keyDown(wrapper.getElement(), { keyCode: KeyCode.down });
+      expect(wrapper.findItemById("item-3")!.getElement()).toHaveFocus();
+    });
+
+    test("Arrow keys move focus in horizontal alignment", () => {
+      const wrapper = renderSupportPromptGroup({ alignment: "horizontal" }, ref);
+
+      ref.current?.focus("item-1");
```

Comment:
Here we should use `!` instead of `?`, so that we get an error message if this value is null unexpectedly, instead of the test silently continuing. Same for the other tests in this file